From 857c2e4407a0f4fcee721372ffed5366bc3051f9 Mon Sep 17 00:00:00 2001
From: Jeremy Allison <jra@samba.org>
Date: Thu, 26 Mar 2009 12:13:28 -0700
Subject: [PATCH] Fix bug #6224 - nmbd waits 5 minutes at startup before checking if it needs to run elections
 Fix logic bug that causes nmbd to wait 5 minutes before
 looking for a master browser. This one is *old* :-). Thanks
 for Simo for bugging me on this.
 Jeremy.

---
 source/nmbd/nmbd_elections.c |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/source/nmbd/nmbd_elections.c b/source/nmbd/nmbd_elections.c
index b50d215..1ed64f4 100644
--- a/source/nmbd/nmbd_elections.c
+++ b/source/nmbd/nmbd_elections.c
@@ -130,9 +130,6 @@ void check_master_browser_exists(time_t t)
 	struct subnet_record *subrec;
 	const char *workgroup_name = lp_workgroup();
 
-	if (!lastrun)
-		lastrun = t;
-
 	if (t < (lastrun + (CHECK_TIME_MST_BROWSE * 60)))
 		return;
 
-- 
1.6.0.6

